DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Utils/ObjectID.cs

```diff
@@ -0,0 +1,41 @@
+using ArcGIS.Desktop.Mapping;
+
+namespace Speckle.Connectors.ArcGIS.Utils;
+
+public struct ObjectID
```

Comment:
I would omit MapMember from this struct, it is confusing why objectID checks for MapMember as below. 

```c#
if (objectId.MapMember == null)
{
   continue;
}
``` 

What I expect from `ObjectID` is just id, if you include MapMember it becomes something else. In this case, we might need another struct that has MapMember too instead considering its nullability on `ObjectID` struct.